src/include/connection.h

```diff
@@ -151,7 +151,17 @@ struct __wt_bucket_storage {
  *  Heuristic controls configuration.
  */
 struct __wt_heuristic_controls {
-    uint32_t obsolete_tw_pages_dirty;
+    /*
+     * Number of btrees processed in the current checkpoint due to obsolete time window information.
+     */
+    uint32_t obsolete_tw_btree_count;
```

Comment:
these should we marked as shared.

Reply:
Updated in da559999eae08a039a972ece429cf1c559d88e29